unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/ChatWidgetHUD/ChatEntry.cs

```diff
@@ -172,10 +199,25 @@ public void Populate(Model chatEntryModel)
         }
     }
 
+    private void PreloadSceneMetadata(ParcelCoordinates parcelCoordinates)
+    {
+        if (MinimapMetadata.GetMetadata().GetSceneInfo(parcelCoordinates.x, parcelCoordinates.y) == null)
+            WebInterface.RequestScenesInfoAroundParcel(new Vector2(parcelCoordinates.x, parcelCoordinates.y), 2);
+    }
+
     public void OnPointerClick(PointerEventData pointerEventData)
     {
         if (pointerEventData.button == PointerEventData.InputButton.Left)
         {
+            int linkIndex = TMP_TextUtilities.FindIntersectingLink(body, pointerEventData.position, null);
+            if (linkIndex != -1)
```

Comment:
I think we can avoid nesting with an early return `if (linkIndex == -1)`.

Reply:
not really because there are things happening after that need to happen even if linkIndex is equal to -1

Reply:
That's true, git cutted me the last part.